roles/openshift_master_facts/lookup_plugins/openshift_master_facts_default_priorities.py

```diff
@@ -91,7 +91,7 @@ def run(self, terms, variables=None, zones_enabled=True, short_version=None,
                 {'name': 'InterPodAffinityPriority', 'weight': 1}
             ])
 
-        if short_version in ['3.5', '3.6']:
+        if short_version in ['3.5', '3.6', '3.7']:
```

Comment:
@sdodson could you point me where are defaults being set in openshift-ansible, so that i can double check?